### DISCUSSION TITLE
想在 Linux 云服务器上用 WebDav 挂载阿里云盘，应该怎么操作

### DISCUSSION THREAD

**Discussion by JooDye** [AlistGo/alist#6605] [Q&A]
如题，只能通过命令行进行。

看文档，好像这个功能是和 alist 本身无关的？
![image](https://github.com/alist-org/alist/assets/33674169/c21c2d36-f90e-4cc2-8c9c-6a31ffbbc3b1)

**Comment by xixka** [comment]
用alist+rclone挂载
